server/lib/models/model.go

```diff
@@ -3,55 +3,50 @@ package models
 import (
 	"log"
 	"fmt"
+	"strings"
 	"database/sql"
 
-	proto "../../proto"
+	proto "github.com/umegaya/channer/server/proto"
+	"github.com/umegaya/channer/server/lib/utils"
 
-	_ "github.com/cockroachdb/cockroach/sql/driver"
+	_ "github.com/cockroachdb/pq"
 	"github.com/umegaya/gorp"
+	"github.com/umegaya/yue"
 )
 
 type Database struct {
 	gorp.DbMap
 	Name string
-	node *Node
+	InsertFixture bool
 }
 type Txn struct {
 	*gorp.Transaction
 }
 type Dbif interface {
-	Delete(list ...interface{}) (int64, error)
-	Exec(query string, args ...interface{}) (sql.Result, error)
-	Get(i interface{}, keys ...interface{}) (interface{}, error)
-	Insert(list ...interface{}) error
-	Prepare(query string) (*sql.Stmt, error)
-	Select(i interface{}, query string, args ...interface{}) ([]interface{}, error)
-	SelectFloat(query string, args ...interface{}) (float64, error)
-	SelectInt(query string, args ...interface{}) (int64, error)
-	SelectNullFloat(query string, args ...interface{}) (sql.NullFloat64, error)
-	SelectNullInt(query string, args ...interface{}) (sql.NullInt64, error)
-	SelectNullStr(query string, args ...interface{}) (sql.NullString, error)
-	SelectOne(holder interface{}, query string, args ...interface{}) error
-	SelectStr(query string, args ...interface{}) (string, error)
-	Update(list ...interface{}) (int64, error)
-	UpdateColumns(filter gorp.ColumnFilter, list ...interface{}) (int64, error)	
+	gorp.SqlExecutor
+	UpdateColumns(filter gorp.ColumnFilter, list ...interface{}) (int64, error)
 }
 
 var dbm Database
 
-func Init(db_addr, certs, host_addr string) error {
-	schema := "http"
+func Init(db_addr, certs, host_addr, data_path string, insert_fixture bool) error {
```

Comment:
__[gometalinter]__
* [golint] exported function Init should have comment or be unexported
* [golint] don't use underscores in Go names; func parameter db_addr should be dbAddr
* [golint] don't use underscores in Go names; func parameter host_addr should be hostAddr
* [golint] don't use underscores in Go names; func parameter data_path should be dataPath
* [golint] don't use underscores in Go names; func parameter insert_fixture should be insertFixture